src/components/templates/Audience/Audience.scss

```diff
@@ -2,142 +2,149 @@
 
 $light: #ffffff;
 
+// If you change this, make sure to also change it in Audience.tsx's SEAT_SIZE
+$seat-size: 4vh;
+$seat-size-min: 25px;
+
+$spacing: calc(1 * min(var(--seat-size), var(--seat-size-min)));
+
 .audience-container {
+  --seat-size: #{$seat-size};
+  --seat-size-min: #{$seat-size-min};
+
   background-attachment: fixed;
   background-position: center 46%;
+
   position: relative;
-  display: flex;
-  flex: 1;
-  -webkit-flex: 1;
-  -ms-flex: 1;
-
-  .video-container {
-    z-index: z(audience-video);
-    position: fixed;
-    left: 50%;
-    top: 28%;
-    width: 45%;
-    top: 50%;
-    left: 50%;
-
-    transform: translate(-50%, -50%);
-
-    .video {
-      position: relative;
-      width: 100%;
-      padding-bottom: 50%;
-      float: left;
-      height: 0;
-
-      .frame {
-        width: 100%;
-        height: 100%;
-        position: absolute;
-        left: 0;
-        border-width: 1px black;
-        border-radius: 15px;
-        box-shadow: 2px 2px 20px $light;
-      }
+  overflow: auto;
+  height: 100%;
+  padding: $spacing;
+  min-width: fit-content;
 
-      .aspect-16-9 {
-        max-width: 88.5%;
-        margin-left: 6.5%;
-      }
```

Comment:
We should probably remove any references of this class from the component if we haven't already.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1237/commits/48743b83a4ffdd69726fa4e1b101f40cc67a636b